README.md

```diff
@@ -114,12 +114,6 @@ You can always override the defaults or config file with CLI flags.
 To get the default config params run `mizu config` <br />
 To generate a new config file with default values use `mizu config -r`
 
-Mizu has several undocumented flags which can be set by using --set flag (e.g., `mizu tap --set dump-logs=true`)
-* **mizu-resources-namespace**: Type - String, See [Namespace-Restricted Mode](#namespace-restricted-mode)
-* **telemetry**: Type - Boolean, Reports telemetry
```

Comment:
Telemetry (in config section)
By default mizu reports usage telemetry, it can be disabled by adding a line of telemtry: false in the <user home dir>/.mizu/config.yaml file